src/Effect.js

```diff
@@ -147,8 +147,15 @@ class Effect extends EventEmitter {
    * @return {number} attribute modified by effect
    */
   modifyAttribute(attrName, currentValue) {
-    const modifier = (this.modifiers.attributes[attrName] || (_ => _)).bind(this);
-    return modifier(currentValue);
+    let modifier = _ => _;
```

Comment:
Ahhh ok I see now. You are setting a default, otherwise if the modifiers attributes is a function you defer to it. Pretty sweet.